src/styles/colors.scss

```diff
@@ -1,19 +1,19 @@
 :root {
-
+    // background-and-text-colors
```

Comment:
1. Remove these comments. They're not saying anything useful, in fact it's not clear at all what they're saying. `// action-colors`, really? Does that comment explain anything?

2. Is there any reason we need to stick to these weird names? If not, let's name them like:
```
--colorA1
--colorA2
--colorA3

--colorB1
--colorB2
--colorB3
```